experiments/modern/src/initialize.js

```diff
@@ -205,18 +205,18 @@ const parsers = {
   },
 };
 
-async function parseChildren(node, zip) {
-  if (node.xmlNode.type === "comment") {
+async function parseChildren(node, children, zip) {
```

Comment:
Its a little odd we are now explicitly passing the xml children in, but before we were pulling them off the xmlNode that we don't have available anymore